### PR TITLE
chore(docs): introduce versioned documentation

### LINKED ISSUE #128: Documentation is versioned
**Is your feature request related to a problem? Please describe.**
There is only one unified documentation, and it is not possible to distinguish between different versions (e. g. 1.10, 1.11, develop).

**Describe the solution you'd like**
A new documentation structure is introduced:
```
/docs/develop/...
/docs/1.11/...
/docs/1.10/...
/docs/index.html => lists all available versions
```

### PR DESCRIPTION
**Pull request by tasso94**
Closes #128
Related to CAM-11745

### CODE REVIEW

File: docs/1.10/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Documentation
+---
+
+# Documentation
+
+The following pages should give an overview how to use the FEEL engine and what FEEL expressions are supported.
```
Comment (koevskinikola) [inline docs/1.10/index.md:7]:
```suggestion
The following pages should give an overview how to use the FEEL engine (v1.10) and what FEEL expressions are supported.
```

File: docs/1.11/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Documentation
+---
+
+# Documentation
+
+The following pages should give an overview how to use the FEEL engine and what FEEL expressions are supported.
```
Comment (koevskinikola) [inline docs/1.11/index.md:7]:
```suggestion
The following pages should give an overview how to use the FEEL engine (v1.11) and what FEEL expressions are supported.
```

File: docs/develop/index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: Documentation
+---
+
+# Documentation
+
+The following pages should give an overview how to use the FEEL engine and what FEEL expressions are supported.
```
Comment (koevskinikola) [inline docs/develop/index.md:7]:
```suggestion
The following pages should give an overview how to use the FEEL engine (develop) and what FEEL expressions are supported.
```

**Comment by tasso94** [comment]
Hey @koevskinikola,

I was thinking exactly the same but decided against it because stating it only on the index page is not helpful if the user continues reading on subpages or stumbles upon a sub-page via google or the forum. I think it should be added automatically to all subpages but this is not in the scope of the ticket. I think it’s a weak compromise to just add it to the index page so I would prefer to do it properly with a separate ticket.

Best,
Tassilo

**Pull request merged**